client/my-sites/plugins/plugin-install-button/index.jsx

```diff
@@ -195,6 +199,38 @@ export class PluginInstallButton extends Component {
 		);
 	}
 
+	renderIncompatiblePluginNotice() {
+		const { translate, isEmbed } = this.props;
+		return (
+			<div className={ classNames( { 'plugin-install-button__install': true, embed: isEmbed } ) }>
+				<span
+					onClick={ this.togglePopover }
+					ref="disabledInfoLabel"
+					className="plugin-install-button__warning"
+				>
+					{ translate( 'Incompatible Plugin' ) }
+				</span>
+				<InfoPopover
+					position="bottom left"
+					popoverName={ 'Plugin Action Disabled Install' }
+					gaEventCategory="Plugins"
```

Comment:
I should probably note, since I don't understand them well, that these are the same as the unreachable notice: https://github.com/Automattic/wp-calypso/blob/804f0eac3a288ec978eed2989a027bdf7ac6e911/client/my-sites/plugins/plugin-install-button/index.jsx#L178-L184